keyboards/eve/meteor/keymaps/via/rules.mk

```diff
@@ -0,0 +1,7 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+
+# build options
+BOOTMAGIC_ENABLE = yes
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = no
```

Comment:
```suggestion
```

Reply:
There are not enough USB interfaces available for MouseKey and ExtraKey options, so both need to be overridden from the base rules.mk (similar to the cftkb mysterium, also a atm32a).